py3status/modules/yandexdisk_status.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Display Yandex.Disk status.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds (10s default)
+    format: prefix text for the Yandex.Disk status
+
+Valid status values include:
+    - Yandex.Disk isn't running!
+    - Error: daemon not started
+    - ...
```

Comment:
This seems irrelevant as the user of this module will not see these.  Maybe just saying you can tell it is running etc in the main docstring.